server/routes/profile.js

```diff
@@ -3,12 +3,12 @@ const router = express.Router();
 const protect = require("../middleware/auth");
 const {
   profileCreatePost,
-  profileUpdatePost,
+  profileUpdatePatch,
```

Comment:
You can organise your files so: `'../controllers/profile/update.js'`. Just a suggestion, though.

Reply:
This sounds like a great idea, I'd need to integrate a few other branches others have made with this change though. If we stumble upon a need we will reformat the code with this in mind and push a hotfix. Thank you!